src/HL/View/Home/Features.hs

```diff
@@ -153,11 +145,10 @@ inference =
 lazy :: Html ()
 lazy =
   do h2_ "Lazy"
-     p_ "Functions don't evaluate their arguments. This means that programs \
-       \can compose together very well, with the ability to write control \
-       \constructs (such as if/else) just by writing normal functions. The purity \
-       \of Haskell code makes it easy to fuse chains of functions together, allowing \
-       \for performance benefits."
+     p_ "Values are computed when they are needed, rather than when they are defined in code. \
+        \ If a value you defined is not used, it never gets evaluated. This means you can spend \
+        \less time thinking about how your program is executed, and more time thinking about what \
+        \your program does."
```

Comment:
I'm not sure this is an improvement. 😮  The original text explains what laziness is the way SPJ explained to a newbie crowd ("functions don't evaluate their arguments"), and covers the benefits of laziness in two more sentences ("composability, control structures" and "fusing for performance"). The suggested replacement here talks about what laziness is, and makes a vague claim out executing a program versus what the program does (I would say this won't mean much to most people, as they sound like the same thing). I'd leave this paragraph as it is until there's a better replacement. It doesn't have to be perfect.